tests/model_fields/test_manytomanyfield.py

```diff
@@ -6,6 +6,13 @@
 
 class ManyToManyFieldTests(SimpleTestCase):
 
+    def test_value_from_object_instance_without_pk(self):
```

Comment:
You might want to use `isolate_apps('model_fields')` to avoid polluting the global `apps` cache.